wcag clarifications may 2026 (#1995)
w3c/wai-website: pages/standards-guidelines/wcag/docs.md

{% include_cached toc.html type="end" %}
{:/}

## WCAG 2 standard

> Web Content Accessibility Guidelines (WCAG) 2 provides a stable, referenceable technical standard. Learn more about WCAG versions from the [[WCAG 2 Overview]](/standards-guidelines/wcag/).

Reviewer: ```suggestion
Web Content Accessibility Guidelines (WCAG) 2 is a stable, referenceable technical standard. Learn more about WCAG versions from the [[WCAG 2 Overview]](/standards-guidelines/wcag/).
```

Not 100% on the change to 'provides'. WCAG *is* a stable, referenceable technical standard so this change seems to add a level of confusion.

When developing web content and web tools, most people will use the documents introduced below, instead of the actual technical standard document.

> **Only the WCAG technical standard includes requirements**, which are called "normative". The supporting materials below are informative; they are not normative. That means the techniques and other supporting materials:

Reviewer 2: ```suggestion
**Only the WCAG technical standard includes requirements**, which are [normative](https://www.w3.org/TR/WCAG22/#interpreting-normative-requirements). The supporting materials below are informative; they are not normative. That means the techniques and other supporting materials:
```
How about linking to the “Interpreting Normative Requirements” section of the standard?

Also, could we more directly say the requirements “are normative” rather than “are called ‘normative’” with the scare quotes?

* do not set or change requirements for WCAG
* are not required for conformance to WCAG

> ## Quick Reference / How to Meet WCAG 2 / WCAG 2 Checklist

Reviewer 2: ```suggestion
## How to Meet WCAG 2 (Quick Reference) checklist
```
Soft editorial tweak to this heading in the spirit of the conversion from title case to sentence case.

Author: Thanks for the suggestion. I'd like to keep "Quick Reference" first for now. Happy to revisit later when we're not trying to get quick updates published. :)

> ## Quick Reference / How to Meet WCAG 2 / WCAG 2 Checklist

Author: ```suggestion
## Quick Reference / How to Meet WCAG 2 / WCAG 2 checklist
```

{% include_cached excol.html type="start" id="diagram" %}

## WCAG 2 documents diagram

{% include_cached excol.html type="middle" %}

The diagram below shows the different WCAG 2 documents for different purposes. **It uses "WCAG 2.0" yet it still applies for WCAG 2.1 and WCAG 2.2.**

_(It is referenced in WCAG 2.0: "The WCAG 2.0 Documents - A diagram and description of how the technical documents are related and linked".)_

{% include image.html src="wcag/wcag20docs.png" alt="" %}

> Not included in the this diagram:

Reviewer 2: ```suggestion
Not included in this diagram:
```
